src/test/java/com/github/wovnio/wovnjava/FileExtensionMatcherTest.java

```diff
@@ -0,0 +1,53 @@
+package com.github.wovnio.wovnjava;
+
+import junit.framework.TestCase;
+
+public class FileExtensionMatcherTest extends TestCase {
+    private FileExtensionMatcher sut;
+
+    protected void setUp() {
+        this.sut = new FileExtensionMatcher();
+    }
+
+    public void testIsFile() {
+        assertIsFile(false, "");
+        assertIsFile(false, "/");
+        assertIsFile(false, "/");
```

Comment:
There is same case.
Original test case had a `null` pattern.

Reply:
> There is same case.
Thanks! Removed now

> Original test case had a null pattern.
Yes, I removed the `null` input handling from the function. It originally begins with
```java
        if (path == null) {	
            return true;	
        }
```
But the input (`HttpServletRequest.getRequestURI()`) cannot be `null`; the servlet context always has a path, even if it is empty string. So we should remove the null-case.